src/App.jsx

```diff
@@ -0,0 +1,26 @@
+import Header  from "./header";
+import Footer from "./Footer";
+import AboutSection from "./aboutSection";
+import ExperienceSection from "./experienceSection";
+import Sidebar from "./sidebar";
+import './style.css'
+
+
+
+function App() {
+  return (
+    <div>
```

Comment:
Please setup and use [prettier ](https://prettier.io/) for formatting the code.

Reply:
fixed!